Add CustomVoiceApi Java sample

baike960 opened the pull request:
I am a Microsoft intern and I wrote a CustomVoiceApi Java sample upon request from a colleague. The code has been tested and could run correctly.

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/Azure-Samples/Cognitive-Speech-TTS?pullRequest=114) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/Azure-Samples/Cognitive-Speech-TTS?pullRequest=114) before we can accept your contribution.<br/><br/>:x: baike960 [sign now](https://cla.opensource.microsoft.com/Azure-Samples/Cognitive-Speech-TTS?pullRequest=114)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/Azure-Samples/Cognitive-Speech-TTS?pullRequest=114) it.</sub>

boltomli commented on CustomVoice-API-Samples/Java/README.md:
```diff
@@ -0,0 +1,15 @@
+## Note:
+
+1.The input text file should be Unicode format with 'UTF-8-BOM' (you can check the text format with your preferred advanced text editor), like the one zh-CN.txt, and should be more than 50 lines.
+1.The voiceId should pick up from MS guys or from the function getVoiceId in the java file.
```
Not sure what do you mean with this note. Who are MS guys? Please kindly elaborate.

baike960 replied:
I will make modifications on Monday. Thanks!

boltomli commented on CustomVoice-API-Samples/Java/src/VoiceSynthesisMain.java:
```diff
@@ -0,0 +1,201 @@
+			localInputTextFile, String locale, String voiceName, String concatenateResult) 
+					throws ApiException, IOException, InterruptedException, JSONException
+    {
+		
```
Please clean up the overall style of the source code files.

baike960 replied:
As for the dependencies, they are organized as jar files located under the lib folder. The jar files are imported through the class path file, so they do not have to be manually imported.

baike960 commented:
Hi Song,
I re-organized the source files and added a pom file to use Maven to automatically download the required files for dependencies. I updated the README.md to record the steps. Please review the changes. Thanks.

baike960 commented:
The coding style was cleaned up

boltomli commented:
@hepower Can you help review the change as well? Thanks.

hepower commented:
it is ok for me. thanks.
